typespec/package.json

```diff
@@ -3,15 +3,15 @@
   "version": "0.1.0",
   "type": "module",
   "dependencies": {
-    "@typespec/compiler": "latest",
-    "@azure-tools/typespec-autorest": "latest",
-    "@azure-tools/typespec-azure-core": "latest",
-    "@azure-tools/typespec-azure-resource-manager": "latest",
-    "@azure-tools/typespec-providerhub": "latest",
-    "@typespec/http": "latest",
-    "@typespec/openapi": "latest",
-    "@typespec/rest": "latest",
-    "@typespec/versioning": "latest"
+    "@typespec/compiler": "^0.60.0",
+    "@azure-tools/typespec-autorest": "^0.46.0",
+    "@azure-tools/typespec-azure-core": "^0.46.0",
+    "@azure-tools/typespec-azure-resource-manager": "^0.46.0",
+    "@azure-tools/typespec-providerhub": "^0.46.0",
+    "@typespec/http": "^0.60.0",
+    "@typespec/openapi": "^0.60.0",
+    "@typespec/rest": "^0.60.0",
+    "@typespec/versioning": "^0.60.0"
```

Comment:
PTAL at the `package-lock.json` to verify. We're still installing the same versions, but the version ranges we consider **allowed** are constrained.